src/main/java/com/todo/Hiplanner/service/MessageServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.todo.Hiplanner.service;
+
+import com.todo.Hiplanner.mapper.MessageMapper;
+import com.todo.Hiplanner.vo.Message;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class MessageServiceImpl implements MessageService{
+
+    private final MessageMapper messageMapper;
+
+
+    @Override
+    public void sendMessage(Message message) {
+        messageMapper.sendMessage(message);
+    }
+
+    @Override
+    public List<Message> selectMessages(Message message) {
+        return messageMapper.selectMessages(message);
+    }
+
+    @Override
+    public Message detailMessage(Message message) {
+        return null;
```

Comment:
구현 예정이시면 별도 주석으로 적어놓으시면 좋을 것 같아요!
intellij / VS code 등 다른 IDE에서 플러그인으로 TODO / FIXME 같은 주석을 스캔해서 바로 확인할 수 있기도 합니다 ㅎㅎ

```suggestion
        // TODO: 메세지 상세 API 서비스 구현 예정
        return null;
```

Reply:
넹 알겠습니다. 코드 작성하고 pr 요청한 줄 알았는데 ... 미처 신경 쓰지 못한 부분까지 .. ㅠㅠ  감사합니다 !